.github/workflows/test-action.yml

```diff
@@ -0,0 +1,29 @@
+name: Print PR Details
```

Comment:
The workflow lacks status checks to verify the script's execution succeeded. Consider adding explicit exit code handling.